src/server/files/internal.js

```diff
@@ -1,4 +1,29 @@
-var parent = window.parent;
-console.log = (...args) => {
-    parent.postMessage(args, 'http://localhost:9000');
+const parent = window.parent;
+
+console.log = (...content) => {
+    logInterceptor('log', content);
+};
+
+console.error = (...content) => {
+    logInterceptor('log--error', content);
+};
+
+console.warn = (...content) => {
+    logInterceptor('log--warn', content);
+};
+
+window.onerror = function (msg) {
+    console.error(msg);
+
+    return false;
+};
+
```

Comment:
There is also console.info which also needs to be added

Reply:
Yeah, sure. For now, for the 1 section, we just need to support `log | error | warning`. The others could be in the future version of the product.

Reply:
Ok